src/backend/commands/createas.c

```diff
@@ -326,7 +328,7 @@ ExecCreateTableAs(ParseState *pstate, CreateTableAsStmt *stmt,
 
 	if (is_matview && into->ivm)
 	{
-		check_ivm_restriction_context ctx = {false, false, false, NIL};
+		check_ivm_restriction_context ctx = {false, false, false, false, NIL,NIL};
```

Comment:
Insert space after comma:
 		check_ivm_restriction_context ctx = {false, false, false, false, NIL, NIL};

Reply:
Thank you for your advice. I will fix it.